### PR TITLE
Makes science fun

### PR DESCRIPTION
**Pull request by lolman360**
## About The Pull Request

ports https://github.com/vgstation-coders/vgstation13/pull/18555. when you finish a game you get (game points/2) rnd points.

## Why It's Good For The Game

it's motherfucking tetris

## Changelog
:cl:
add: tetris rnd. You can print the TETRIS machine roundstart.
imageadd: added tetris icons
/:cl:

### CODE REVIEW

**Comment by Tupinambis** [comment]
This is probably not balanced, but honestly I'd play science if it was Tetris so 👍

**Comment by Trilbyspaceclone** [comment]
Were the poly normals?

**Comment by lolman360** [comment]
if you exploit this i'm banning you from life

**Comment by Linzolle** [comment]
finally, my skills are a tetris grand master come into play

**Comment by ghost** [comment]
@lolman360 you forgot to actually add a way to get it in-game

**Comment by lolman360** [comment]
@kappa-sama i swear i added it to techwebs hol up

**Comment by lolman360** [comment]
i didnt push it lmao (

**Comment by RogueSteampunker** [comment]
inb4 kevinz comes in to ruin all our dreams

**Comment by silicons** [comment]
hi i'm here to ruin everyone's dreams

yeah this is a glaring security hole sorry
i would like a server side implemnetation of tetris but i'm hard no on this for that reason

i'm fine with it staying testmerged for.. whatever time y'all want though :) hell testmerge this for a year idgaf lmao just please no href exploits in my code :(

**Comment by lolman360** [comment]
)

File: code/modules/research/machinery/tetris.dm [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+	<!DOCTYPE HTML PUBLIC '-//W3C//DTD HTML 4.01 Transitional//EN'>
+	<html><head>
+	<META NAME='description' content='Tetris is a PC-Game, which is a part of the '7 by one stroke' package, written in HTML and JavaScript'>
+	<META NAME='author' content='Lutz Tautenhahn'>
+	<META NAME='keywords' content='Game, Tetris, Streich, Stroke, JavaScript'>
+	<META HTTP-EQUIV='Content-Type' CONTENT='text/html; charset=iso-8859-1'>
+	<title>Telemetry Enhanced Testing and Research Informatic Simulator</title>
+	<script language='JavaScript'>
+	function submitScore(s){
+		window.location.href = 'byond://?src=\ref[src];tetrisScore=' + s;
```
Comment (LetterN) [inline code/modules/research/machinery/tetris.dm:51]: kek

**Pull request closed without merging**